### PR TITLE
eve/schema: allow authorities in dns.answers in alert

### PR DESCRIPTION
**Pull request by jasonish**
While this might not be ideal, it is how things are currently logged.

### CODE REVIEW

File: etc/schema.json [hunk 1]
```diff
@@ -1156,7 +1156,12 @@
                         "opcode": {
                             "description": "DNS opcode as an integer",
                             "type": "integer"
-                        }
+                        },
+			"authorities": {
+			    "description": "Array of authorities in DNS response",
+			    "$comment": "Probably also need answers and additional records",
```
[the remark this answers is not in the document]
Comment (jasonish) [reply etc/schema.json:1162]: Yeah, somewhat of a todo of what might also pop up in addition to authorities. But that's another task and for now we have to unblock authorities from holding up https://github.com/OISF/suricata/pull/10126.

File: etc/schema.json [hunk 2]
```diff
@@ -1156,7 +1156,12 @@
...
+			    "type": "array"
```
Comment (catenacyber) [inline etc/schema.json:1163]: I think this should be completed by the list of items and `"additionalProperties": false`

I think we do not want quick and dirty green CI

File: etc/schema.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (victorjulien) [reply etc/schema.json:1162]: Lets not use this file as a todo list. If more work is to be done, we can track it in tickets.

File: etc/schema.json (hunk 2 above)
Comment (jasonish) [reply etc/schema.json:1163]: My purpose is to unblock some work before getting to the proper fixes.

**Comment by jasonish** [comment]
https://github.com/OISF/suricata/pull/10144

**Pull request closed without merging**